vendor/modules.txt

```diff
@@ -44,6 +44,8 @@ github.com/gofrs/uuid/v5
 github.com/golang/mock/gomock
 # github.com/google/go-cmp v0.6.0
 ## explicit; go 1.13
+# github.com/google/uuid v1.1.2
```

Comment:
Why is this being added to vendor/modules, I dont see this being added as an import? Can you check if `go mod tidy` and `go mod vendor` removes it?